mmv1/products/integrationconnectors/Connection.yaml

```diff
@@ -0,0 +1,1120 @@
+          - auth_config.0.oauth2_jwt_bearer
+          - auth_config.0.oauth2_client_credentials
+          - auth_config.0.ssh_public_key
+          - auth_config.0.oauth2_auth_code_flow
+      - !ruby/object:Api::Type::NestedObject
+        name: "oauth2ClientCredentials"
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "clientId"
+            description: |
+              Secret version of Password for Authentication.
+          - !ruby/object:Api::Type::NestedObject
+            name: "clientSecret"
+            description: |
+              Secret version reference containing the client secret.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: "secretVersion"
+                required: true
+                description: |
+                  The resource name of the secret version in the format,
+                  format as: projects/*/secrets/*/versions/*.
+        exactly_one_of:
+          - auth_config.0.user_password
+          - auth_config.0.oauth2_jwt_bearer
+          - auth_config.0.oauth2_client_credentials
+          - auth_config.0.ssh_public_key
+          - auth_config.0.oauth2_auth_code_flow
+      - !ruby/object:Api::Type::NestedObject
+        name: "sshPublicKey"
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "username"
+            description: |
+              The user account used to authenticate.
+          - !ruby/object:Api::Type::NestedObject
+            name: "sshClientCert"
+            description: |
+              Secret version reference containing the client secret.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: "secretVersion"
+                required: true
+                description: |
+                  The resource name of the secret version in the format,
+                  format as: projects/*/secrets/*/versions/*.
+          - !ruby/object:Api::Type::String
+            name: "certType"
+            description: |
+              Format of SSH Client cert.
+          - !ruby/object:Api::Type::NestedObject
+            name: "sshClientCertPass"
+            description: |
+              Secret version reference containing the client secret.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: "secretVersion"
+                required: true
+                description: |
+                  The resource name of the secret version in the format,
+                  format as: projects/*/secrets/*/versions/*.
+        exactly_one_of:
+          - auth_config.0.user_password
+          - auth_config.0.oauth2_jwt_bearer
+          - auth_config.0.oauth2_client_credentials
+          - auth_config.0.ssh_public_key
+          - auth_config.0.oauth2_auth_code_flow
+      - !ruby/object:Api::Type::NestedObject
+        name: "oauth2AuthCodeFlow"
+        properties:
```

Comment:
mark one of the subfields required, or have at_least_one_of

Reply:
Added